### PR TITLE
removed ul-li, fixed indentation

### PR DESCRIPTION
**Pull request by 81jumpstr**

### CODE REVIEW

File: page-popupshop.php [hunk 1]
```diff
@@ -29,15 +25,11 @@
...
+          </article>
+    <?php 
         endwhile;
-      ?>
-        </ul>
-    <?php
       endif;
     ?>
-
 	</div>
```
Comment (royscheeren) [inline page-popupshop.php]: Indentation is off

File: page-popupshop.php [hunk 2]
```diff
@@ -29,15 +25,11 @@
               <button href="#" class="intervention">Events</button>
               <button href="#" class="workshop">Setupshop</button>
             </div>
```
Comment (royscheeren) [inline page-popupshop.php]: Why wrap it in a div? Theres no class so you don't seem to be styling it.

**Review by royscheeren — CHANGES_REQUESTED** [review]
Looks good if you dont

**Review by royscheeren — APPROVED** [review]
Look how nice and clean this looks

**Comment by 81jumpstr** [comment]
lol ...yes, it does.

**Pull request merged**